Week3/homework/4-step3.js

```diff
@@ -1,2 +1,7 @@
 'use strict';
 // paste your freeCodeCamp solutions in here
+function Dog () {
+    this.name = "Jumbo";
+    this.color = "Black";
+    this.numLegs = 4;
+    }
```

Comment:
Correct. ✅ 
Just wondering why you removed the line that actually creates an instance of a `Dog`.